classes/log/file.php

```diff
@@ -72,7 +72,7 @@ public static function write($level, $msg, $method = null)
 	
 						//
 						if($level === 'Error'){
-							if ($level == 'Error') var_dump($backtrace);
+							if ($level == 'Error') //var_dump($backtrace);
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Inline control structures are not allowed